kani-compiler/src/main.rs

```diff
@@ -118,10 +140,14 @@ impl Callbacks for KaniCallbacks {}
 
 /// Generate the arguments to pass to rustc_driver.
 fn generate_rustc_args(args: &ArgMatches) -> Vec<String> {
-    let mut gotoc_args =
-        rustc_gotoc_flags(args.value_of(parser::KANI_LIB).unwrap_or(std::env!("KANI_LIB_PATH")));
+    // TODO: Re-order funcs inside this file
```

Comment:
I'm tempted to move all this stuff out of the main file. Out of the scope of this change though. :)